tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/remove_non_matching_participant_test.py

```diff
@@ -0,0 +1,324 @@
+                ],
+                'loaded_ids': [
+                    101, 102, 103, 104, 201, 202, 203, 204, 301, 302, 303, 304,
+                    401, 402, 403, 404
+                ],
+                'sandboxed_ids': [204, 303, 304],
+                'cleaned_values': [
+                    (101, 0, 1991, 0, 0),
+                    (102, 0, 1992, 0, 0),
+                    (103, 0, 1993, 0, 0),
+                    (104, 0, 1994, 0, 0),
+                    (201, 0, 1991, 0, 0),
+                    (202, 0, 1992, 0, 0),
+                    (203, 0, 1993, 0, 0),
+                    (301, 0, 1991, 0, 0),
+                    (302, 0, 1992, 0, 0),
+                    (401, 0, 1991, 0, 0),
+                    (402, 0, 1992, 0, 0),
+                    (403, 0, 1993, 0, 0),
+                    (404, 0, 1994, 0, 0),
+                ]
+            },
+            {
+                'name':
+                    f'{OBSERVATION}',
+                'fq_table_name':
+                    f'{self.project_id}.{self.dataset_id}.{OBSERVATION}',
+                'fq_sandbox_table_name':
+                    f'{self.project_id}.{self.sandbox_id}.{OBSERVATION}_{"_".join(self.rule_instance.issue_numbers)}',
+                'fields': [
+                    'observation_id', 'person_id', 'observation_concept_id',
+                    'observation_date', 'observation_type_concept_id'
+                ],
+                'loaded_ids': [
+                    1001, 1002, 1003, 1004, 2001, 2002, 2003, 2004, 3001, 3002,
+                    3003, 3004, 4001, 4002, 4003, 4004
+                ],
+                'sandboxed_ids': [2004, 3003, 3004],
+                'cleaned_values': [
+                    (1001, 101, 0, parse('2022-01-01').date(), 0),
+                    (1002, 102, 0, parse('2022-01-02').date(), 0),
+                    (1003, 103, 0, parse('2022-01-03').date(), 0),
+                    (1004, 104, 0, parse('2022-01-04').date(), 0),
+                    (2001, 201, 0, parse('2022-01-01').date(), 0),
+                    (2002, 202, 0, parse('2022-01-02').date(), 0),
+                    (2003, 203, 0, parse('2022-01-03').date(), 0),
+                    (3001, 301, 0, parse('2022-01-01').date(), 0),
+                    (3002, 302, 0, parse('2022-01-02').date(), 0),
+                    (4001, 401, 0, parse('2022-01-01').date(), 0),
+                    (4002, 402, 0, parse('2022-01-02').date(), 0),
+                    (4003, 403, 0, parse('2022-01-03').date(), 0),
+                    (4004, 404, 0, parse('2022-01-04').date(), 0),
+                ]
+            },
+        ]
+
+        self.default_test(tables_and_counts)
+
+    def tearDown(self):
+        pass
+
+    @classmethod
+    def tearDownClass(cls):
+        test_util.delete_all_tables(cls.dataset_id)
+        test_util.delete_all_tables(cls.ehr_dataset_id)
+        test_util.delete_all_tables(cls.validation_dataset_id)
+        test_util.delete_all_tables(cls.sandbox_id)
```

Comment:
I hope we can get rid of these and use a custom method added to the client, but that's for later.